doc/bips.md

```diff
@@ -24,6 +26,7 @@ BIPs that are implemented by Bitcoin Core (up-to-date up to **v0.21.0**):
   Support can be optionally disabled at build time since **v0.18.0** ([PR 14451](https://github.com/bitcoin/bitcoin/pull/14451)),
   and it is disabled by default at build time since **v0.19.0** ([PR #15584](https://github.com/bitcoin/bitcoin/pull/15584)).
   It has been removed as of **v0.20.0** ([PR 17165](https://github.com/bitcoin/bitcoin/pull/17165)).
+* [`BIP 84`](https://github.com/bitcoin/bips/blob/master/bip-0049.mediawiki): The experimental descriptor wallets introduced in **v0.21.0** by default use the Hierarchical Deterministic Wallet derivation proposed by BIP 84.
```

Comment:
* [`BIP 84`](https://github.com/bitcoin/bips/blob/master/bip-0049.mediawiki): The experimental descriptor wallets introduced in **v0.21.0** by default use the Hierarchical Deterministic Wallet derivation proposed by BIP 84 [(PR #16528)](https://github.com/bitcoin/bitcoin/pull/16528).

Reply:
@luke-jr This is actually linking to BIP 49, not to BIP 84.